doc/TeamsConnector.md

```diff
@@ -0,0 +1,81 @@
+# reshuffle-microsoft-connectors
+
+[Code](https://github.com/reshufflehq/reshuffle-microsoft-connectors) |
+[npm](https://www.npmjs.com/package/reshuffle-microsoft-connectors) |
+[Code sample](https://github.com/reshufflehq/reshuffle/tree/master/examples/microsoft/teams)
+
+`npm install reshuffle-microsoft-connectors`
+
+### Reshuffle Microsoft Connectors
+
+This package contains a collection of [Reshuffle](https://github.com/reshufflehq/reshuffle)
+connectors to [Microsoft](https://microsoft.com).
```

Comment:
I think for the connector-specific docs, we should follow the AWS connectors' pattern where we use the term connector. E.g. 
`This Reshuffle connector can be used to access AWS Lambda. It is implemented using Amazon's Lambda SDK.`

I know the Excel readme does not follow this pattern. I will submit a PR